### DISCUSSION TITLE
Yandex metrica 403 - help

### DISCUSSION THREAD

**Discussion by Me-atom** [getredash/redash#7238] [Q&A]
Good day. I'm a newbie. I deployed Redash and connected to Yandex Metrica. The test request goes through without errors, but the working request constantly returns a 403 response. At the same time, a get request through a browser with the same token does not return errors. I can't figure out what the problem could be. I would also be glad to receive advice on working with metrics from those who have already done something similar.

Error running query: Code: 403, message: {"errors":[{"error_type":"access_denied","message":"Access is denied"}],"code":403,"message":"Access is denied"}

////////////////////////////////////////////////

Добрый день. Я новичок. Развернул Редаш подключаюсь к яндекс метрике. Тестовый запрос проходит без ошибок, но рабочий запрос постоянно выдаёт 403 ответ. При этом гет запрос через браузер с тем же токеном ошибок не выдаёт. Не могу понять в чём может быть проблема. Так же буду рад получить советы по работе с метрикой от тех, кто уже делал подобное.

Error running query: Code: 403, message: {"errors":[{"error_type":"access_denied","message":"Access is denied"}],"code":403,"message":"Access is denied"}

**Comment by biakaveron** [comment]
Скорее всего тело запроса неправильное. В браузере может работать, например, потому что не в инкогнито запускаешь (и влияет авторизация в Яндексе). Лучше проверять специальными инструментами, например Постманом. 

Навскидку - перепроверь, что ты в теле запроса передаешь параметр ids с корректным номером счетчика
